man/systemd-system.conf.xml

```diff
@@ -322,6 +322,11 @@
         The <literal>:</literal> means variables in the file can reference other environment
         variables, using <literal>${VARNAME}</literal> expansion.
         </para>
+
+        <para><varname>DefaultEnvironmentFile=</varname> defaults to empty for
+        system sessions, and defaults to <filename>/etc/environment</filename>,
+        <filename>/usr/lib/environment.d/*.env</filename>, and
+        <filename>~/.config/environment.d</filename> for user sessions.
```

Comment:
> might need to be a bit longer, including /usr/local/lib/environment.d/

The general principle used in at least Debian is that if a search path includes `/usr/foo` (for software installed via the distribution) then it should probably also include `/usr/local/foo` (for software installed by the sysadmin). `XDG_DATA_DIRS` and various systemd-originated search paths follow that general principle fairly consistently, and I'd recommend it.